server/svix-server/src/core/types.rs

```diff
@@ -577,4 +614,46 @@ mod tests {
         let evt_name = EventChannel(long_str);
         assert!(evt_name.validate().is_err());
     }
+
+    #[test]
+    fn test_endpoint_headers_validation() {
+        let hdr_map = HashMap::from([
+            ("valid".to_owned(), "true".to_owned()),
+            ("also-valid".to_owned(), "true".to_owned()),
+        ]);
+        let endpoint_headers = EndpointHeaders(hdr_map);
+        assert!(endpoint_headers.validate().is_ok());
```

Comment:
Better to just unwrap instead of `is_ok()` because then you'll get the actual error printed if it fails.

Reply:
Fixed.